qcelemental/covalent_radii.py

```diff
@@ -0,0 +1,148 @@
+"""
+Contains covalent radii
+"""
+
+import collections
+from decimal import Decimal
+
+from . import datum
+from .periodic_table import periodictable
+
+from .exceptions import NotAnElementError, DataUnavailableError
+
+
+class CovalentRadii:
+    """Covalent radii sets.
+
+    Parameters
+    ----------
+    context : {'ALVAREZ2008'}
+        Origin of loaded data.
+
+    Attributes
+    ----------
+    name : str
+        The name of the context ('ALVAREZ2008')
+    cr : dict of Datum
+        Each covalent radius is an entry in `cr`, where key is the
+        "Fe"-cased element symbol if generic or symbol-prefixed label
+        if specialized within element. The value is a Datum object with
+        `lbl` the same as key, `units`, `data` value as Decimal object,
+        and any uncertainty in the `comment` field.
+    year : int
+        The year the context was created.
+
+    """
+
+    def __init__(self, context="ALVAREZ2008"):
+        self.cr = collections.OrderedDict()
+
+        from .data import alvarez_2008_covalent_radii
+
+        if context == "ALVAREZ2008":
+            self.doi = alvarez_2008_covalent_radii["doi"]
+            self.native_units = alvarez_2008_covalent_radii["units"]
+
+            for cr in alvarez_2008_covalent_radii["covalent_radii"]:
+                self.cr[cr[0]] = datum.Datum(cr[0], self.native_units, Decimal(cr[1]), cr[2], doi=self.doi)
+        else:
+            raise KeyError("Context set as '{}', only contexts {'ALVAREZ2008', } are currently supported")
+
+        self.name = context
+        self.year = int(alvarez_2008_covalent_radii["date"][:4])
+
+        # Extra relationships
+        aliases = [
+            ('C',  'angstrom', self.cr['C_sp3'].data,       'Largest (sp3) chosen for generic atom'),
+            ('Mn', 'angstrom', self.cr['Mn_highspin'].data, 'Larger (high-spin) chosen for generic atom'),
+            ('Fe', 'angstrom', self.cr['Fe_highspin'].data, 'Larger (high-spin) chosen for generic atom'),
+            ('Co', 'angstrom', self.cr['Co_highspin'].data, 'Larger (high-spin) chosen for generic atom'),
+        ]  # yapf: disable
+
+        # add alternate names to help QC programs
+        for alias in aliases:
+            ident, units, value, comment = alias
+            self.cr[ident.capitalize()] = datum.Datum(ident, units, value, comment)
+
+    def __str__(self):
+        return "CovalentRadii(context='{}')".format(self.name)
+
+    def get(self, atom, return_tuple=False, units=None):
```

Comment:
Would be much happier if default units was `bohr` so that we can bank on this as the default.

Reply:
Do you want the units arg here to default to "bohr" instead of "natural"? I'm good with that so long as the internal (in Decimal; read from table) can remain Ang.

This `pint` is going to drag our unit abbreviations into the mainstream. I already learned `Dyne`-->`dyn`.

Reply:
Yes please on the `Bohr`. Happy to have internal remain as Decimal and even perhaps the tuple return in Ang as well.  Yes, pint will straighten us out with a singular definition of abbreviations (a good thing).